src/stactools/naip/stac.py

```diff
@@ -102,7 +106,8 @@ def create_item(state,
                               precision=6)
 
     if fgdc_metadata_href is not None:
-        fgdc_metadata_text = pystac.STAC_IO.read_text(fgdc_metadata_href)
+        stac_io = DefaultStacIO()
+        fgdc_metadata_text = stac_io.read_text(fgdc_metadata_href)
```

Comment:
Why not use read_text here from stactools core?

Reply:
Yup, good call, will fix.